iceoryx_examples/icediscovery/README.md

```diff
@@ -46,7 +46,7 @@ It is included via:
 On that object we can call the method `findService` which expects the three
 service [string identifiers](https://github.com/eclipse-iceoryx/iceoryx/blob/v2.0.0/doc/website/getting-started/overview.md#creating-service-descriptions-for-topics)
 and a callable which will be applied to all matching services.
-In addition we have to specify whether we want to search for publishers (`MessagingPattern::PUB_SUB`)
+In addition, we have to specify whether we want to search for publishers (`MessagingPattern::PUB_SUB`)
```

Comment:
Those commas are not needed (not wrong, but also not usually used unless we want to really emphasize what follows). Same for many others.

Whatever.